mantle/README.md

```diff
@@ -350,6 +350,30 @@ for more information about the `.boto` file.
 }
 ```
 
+### ibmcloud
+`ibmcloud` uses `~/.ibmcloud/apikey.json`. This can be populated by downloading the apikey from the IBMCloud UI (https://cloud.ibm.com/login) or by using the IBMCloud cli (https://cloud.ibm.com/docs/cli?topic=cli-install-ibmcloud-cli). This would require the user to login with the correct credentials:
+```
+$ ibmcloud login --sso
+```
+or by using an existing apikey:
+```
+$ ibmcloud login --apikey <user-api-key>
+```
+Once logged in, an api key can be created:
+```
+ $ ibmcloud iam api-key-create other_key --file ~/.ibmcloud/apikey.json --output json
```

Comment:
just to cover all bases, and maybe @rpsene or @bpradipt can explain, but when using my version of `ibmcloud` (ibmcloud version 1.5.1+7684ebe-2021-04-27T18:17:58+00:00) -- the dir I end up with after running `ibmcloud login -sso` is  `~/.bluemix/`.  If I run this command, I get the error that the dir doesn't exist. 
```
ibmcloud iam api-key-create mykey --file ~/.ibmcloud/apikey.json --output json
FAILED
open /home/christy/.ibmcloud/apikey.json: no such file or directory
```